README.md

```diff
@@ -1,2 +1,86 @@
-# Fire-Extinguisher-Bot
-Open Projects 2021
+# **Fire-Extinguisher-Bot**
+*Open Projects 2021*
+![Bot Design](Images_videos/Images/Bot_view_1.jpg)
+<div align="center"><em>Bot View 1</em></div>
+<br/>
+
+![Bot Design 2](Images_videos/Images/Bot_view_4.PNG)
+<div align="center"><em>Bot View 2</em></div>
+<br/>
+
+## **Abstract**
+- Automatic Fire Extinguisher Robot is a Hardware-based model used to put out the fire during fire accidents automatically. The Bot has features to move in the direction of fire  and extinguish it automatically.
```

Comment:
Justify all the text using paragraph tags. For eg.
`<p align="justify">`
  `<h2>Abstract</h2>`
  `<p>A Rube Goldberg Machine is a contraption intentionally designed to perform a simple task in an indirect and overly complicated way. The aim of this project is to depict the life of a college student through a similar complex assembly of mechanical and electronic components linked with each other creating a metaphor between humans and machines.</p>`
`</p>`